resources/service-catalog/charts/catalog/values.yaml

```diff
@@ -1,134 +1,60 @@
 # Default values for Service Catalog
 # service-catalog image to use
-image: quay.io/kubernetes-service-catalog/service-catalog:v0.1.43
+#TODO: change that!
+image: awalach/service-catalog:crds
```

Comment:
"change that" when?

Reply:
When we'll have new SC release. Possibly before merging this PR :)

Reply:
Changed